index.d.ts

```diff
@@ -0,0 +1,242 @@
+
+	@default {}
+	*/
+	headers?: Headers;
+
+	/**
+	Set a custom [user agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent).
+	*/
+	userAgent?: string;
+
+	/**
+	Set cookies in [browser string format](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) or [object format](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetcookiecookies).
+
+	Tip: Go to the website you want a cookie for and [copy-paste it from DevTools](https://stackoverflow.com/a/24961735/64949).
+	*/
+	cookies?: (string | SetCookie)[];
+
+	/**
+	Credentials for [HTTP authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication).
+	*/
+	authentication?: Authentication;
+
+	/**
+	The specified function is called right before the screenshot is captured. It gives you a lot of power to do custom stuff. The function can be async.
+
+	Note: Make sure to not call `page.close()` or `browser.close()`.
+	*/
+	beforeScreenshot?: BeforeScreenshot;
+
+	/**
+	Show the browser window so you can see what it's doing, redirect page console output to the terminal, and slow down each Puppeteer operation.
+
+	Note: This overrides `launchOptions` with `{headless: false, slowMo: 100}`.
+
+	@default false
+	*/
+	debug?: boolean;
+
+	/**
+	Options passed to [`puppeteer.launch()`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions).
+
+	Note: Some of the launch options are overridden by the `debug` option.
+
+	@default {}
+	*/
+	launchOptions?: LaunchOptions;
+
+	/**
+	Overwrite the destination file if it exists instead of throwing an error.
+
+	This option applies only to `captureWebsite.file()`.
+
+	@default false
+	*/
+	overwrite?: boolean;
+}
+
+/**
+Devices supported by the `emulateDevice` option.
+*/
+export const devices: string[];
+
+/**
+Capture a screnshot of the given `url` and save it to the given `outputFilePath`.
+
+@param url - The URL, file URL, data URL, or local file path to the website.
+@param outputFilePath - The path to write the screenshot.
+@returns A promise that resolves when the screenshot is written to the given file path.
+
+@example
+```
+import captureWebsite from 'capture-website';
+
+(async () => {
+	await captureWebsite.file('https://sindresorhus.com', 'screenshot.png');
+})();
+```
+*/
+export function file(
+	url: string,
+	outputFilePath: string,
+	options?: Options
+): Promise<void>;
+
+/**
+Capture a screnshot of the given `url`.
+
+@param url - The URL, file URL, data URL, or local file path to the website.
+@returns The screenshot as binary.
+*/
+export function buffer(url: string, options?: Options): Promise<Buffer>;
+
+/**
+Capture a screnshot of the given `url`.
+
+@param url - The URL, file URL, data URL, or local file path to the website.
+@returns The screenshot as [Base64](https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding).
+*/
+export function base64(url: string, options?: Options): Promise<string>;
```

Comment:
The functions should not be named export, but rather properties on the default export, which is `captureWebsite`.

Like this: https://github.com/sindresorhus/npm-keyword/blob/db7e0b7ff3c90eb49a6d519d17b054b4af600bcf/index.d.ts#L15-L41